libnmstate/nm/bridge.py

```diff
@@ -17,20 +17,23 @@
 # along with this program. If not, see <https://www.gnu.org/licenses/>.
 #
 
-from libnmstate.error import NmstateNotImplementedError
 from libnmstate.nm import connection
 from libnmstate.nm import nmclient
+from libnmstate.nm.bridge_port_vlan import PortVlanFilter
 from libnmstate.schema import LinuxBridge as LB
 
 
 BRIDGE_TYPE = "bridge"
 
 
-def create_setting(options, base_con_profile):
+def create_setting(options, base_con_profile, enable_vlan_filtering=False):
     bridge_setting = _get_current_bridge_setting(base_con_profile)
     if not bridge_setting:
         bridge_setting = nmclient.NM.SettingBridge.new()
+    if not enable_vlan_filtering:
+        enable_vlan_filtering = bridge_setting.props.vlan_filtering
 
+    bridge_setting.props.vlan_filtering = enable_vlan_filtering
```

Comment:
Okay, my brain have halted thinking on this and sorry if I am missing something but this is the same than: 

```
if enable_vlan_filtering:
    bridge_setting.props.vlan_filtering = enable_vlan_filtering
```

or 

```
bridge_setting.props.vlan_filtering = bridge_setting.props.vlan_filtering or enable_vlan_filtering
```

And I think you should do directly:
```
bridge_setting.props.vlan_filtering = enable_vlan_filtering
```
What do you think? Thanks!

Reply:
This is confusing because the decision is not taken in one place.
Are the ports properties reconfigured from scratch here? I mean, by the time the desired state reaches this place, does it include all ports? If so, the decision is limited to the desired state and it does not matter what existed.
In case the desired state only includes a portion of the ports and all the rest are not touched, then how those are getting removed and is the desired state specify it. In such a case you'll need to pass through all non-removed ports and check if any have a vlan, if all do not, then maybe you need to remove the vlan-filtering.

Reply:
Stopped exposing a parameter dedicated to this.